### PR TITLE
Remove 'Console.WriteLine' calls in code

### PR DESCRIPTION
**Pull request by speyck**
### 🤔 This is a ...

- [ ] New feature
- [X] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

#2161 #2210 

### 💡 Background and solution

Code contained a lot of `Console.WriteLine` calls, which are all only used for debugging. They should not leave the development environment. Other than that, they should be replaced by injecting `ILogger` and using the `LogDebug` method on it. `Console.WriteLine` is not supported on all platforms which could lead to issues.

I removed all `Console.WriteLine` calls (or commented them out if I thought they would be used later again). I did not implement any `ILogger`, if it's necessary later it can still be added.
### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | X          |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=2268) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=2268) before we can accept your contribution.<br/><br/>:x: speyck [sign now](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=2268)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/ant-design-blazor/ant-design-blazor?pullRequest=2268) it.</sub>

File: components/menu/SubMenu.razor.cs [hunk 1]
```diff
@@ -87,7 +87,7 @@ private void SetClass()
                         var eval = RootMenu?.InternalMode == MenuMode.Inline && IsOpen;
                         if (Key == "sub1")
                         {
-                            Console.WriteLine(eval);
+                            //Console.WriteLine(eval);
                         }
```
Comment (ElderJames) [inline components/menu/SubMenu.razor.cs:91]: Please delete this block.

Comment (speyck) [reply components/menu/SubMenu.razor.cs:91]: I also commented out some other calls which had some more text in them. Should I just remove all the calls also the ones that are commented out?

Comment (speyck) [reply components/menu/SubMenu.razor.cs]: @ElderJames updated.

**Review by ElderJames — APPROVED** [review]

**Comment by speyck** [comment]
@ElderJames you might want to close the related issue: https://github.com/ant-design-blazor/ant-design-blazor/issues/2161 https://github.com/ant-design-blazor/ant-design-blazor/issues/2210

**Comment by ElderJames** [comment]
Oh yes!

**Pull request merged**